src/ui/components/MigrateToWorkbookDialog/MigrateToWorkbookDialog.tsx

```diff
@@ -5,14 +5,22 @@ import {Dialog} from '@gravity-ui/uikit';
 import block from 'bem-cn-lite';
 import {I18n} from 'i18n';
 import {useDispatch, useSelector} from 'react-redux';
+import {ConnectorType} from 'shared';
 import {ErrorContent} from 'ui/index';
+import {getRelations} from 'ui/store/actions/copyEntriesToWorkbook';
```

Comment:
Would be better not to use actions and selectors from another store. The `copyEntriesToWorkbook` store was created for copying entries between workbooks dialog. Can we duplicate `getRelations` and selectors to `migrationToWorkbook` store?